src/components/Layout/Header/DegradedStateBanner.tsx

```diff
@@ -37,6 +42,41 @@ export const DegradedStateBanner = memo(() => {
   const buttonBg = useColorModeValue('blackAlpha.100', 'whiteAlpha.100')
   const assets = useSelector(selectAssets)
   const erroredAccountIds = useSelector(selectPortfolioErroredAccountIds)
+  const { degradedChainIds, isFetching: isDiscoverAccountsFetching } = useDiscoverAccounts()
+  const { isSnapInstalled } = useIsSnapInstalled()
+  const {
+    state: { deviceId },
+  } = useWallet()
+  const queryClient = useQueryClient()
+
+  const erroredChainIds = useMemo(() => {
+    const erroredChains = uniq(
+      degradedChainIds.reduce(
```

Comment:
with `undefined` filtered out, we can clean this up with just `.map` and suggestion to use `chainIdToFeeAssetId`.

```
    const erroredChains = uniq(
      degradedChainIds.map(chainId => {
        const asset = assets[chainIdToFeeAssetId(chainId)]

        return {
          name: asset?.networkName ?? asset?.name,
          icon: asset?.networkIcon ?? asset?.icon,
        }
      }),
    )
```

Reply:
Not sure to understand this one as `erroredChains` is merging both `erroredAccounts` from RTK and `erroredChainIds` from `useDiscoverAccounts`

Reply:
Closing to avoid getting this stale, if we want to change something, don't hesitate to ping me for follow ups